source/_integrations/shelly.markdown

```diff
@@ -113,6 +113,11 @@ Not all devices support all input events. You can check on [Shelly API Reference
 
 </div>
 
+## Silabs based powered devices (CoIoT)
+For Silabs based devices such as the Shelly Motion it is required to enable "CoIoT" in the device for it to work with Home Assistant.
+To setup "CoIoT" on your Shelly device: Navigate to the local IP-address of your Shelly device, "Internet & Security", "COIOT", check the box, enter your local Home Assistant IP-Address in the "remote address" field and push "save".
```

Comment:
```suggestion
To setup "CoIoT" on your Shelly device: Navigate to the local IP-address of your Shelly device, "Internet & Security", "COIOT", check the box, enter your local Home Assistant IP-Address in the "remote address" field and push "save".

```
Need a blank line else the image and the text are mixed.
![image](https://user-images.githubusercontent.com/1858925/109674962-054e9b00-7b80-11eb-8ad4-8dd1e9dfef67.png)